script/Arcueid/Day 1/01_00_ARC01_5.txt

```diff
@@ -307,7 +307,7 @@ The girl tilts her head gently as if puzzled by my confused attitude.
 [sha:603bcfce7122f507383934f733a5afdf815a831e]{
 -- Page 269, Offset 1843.
 -- わたし、また間違えてしまったのかと恐くなってしまったじゃないですか」
-You had me terrified that I might have gotten the wrong person!"
+You had me terrified that I might have gotten the wrong person again!"
```

Comment:
May actually want to leave off the また just because of how we've naturalized 間違えた, or go back to a more literal "Oh no, I've made another mistake!" if the また is that important. The original sentence doesn't really suggest that she's bringing random people back to the house like this does.

Reply:
Yeah, that probably isn't the intent. I do think that the また should be included though, cos its her first scene that immediately characterises her as a 'klutz'. Maybe something like 'You had me terrified that I made **another** mistake.'

Reply:
The intent of this line is foreshadowing for the other SHIKI.

Reply:
Does that mean you have an opinion one way or the other on how it reads?

Reply:
I think the way it is written (with again) is totally fine